CommonProjectOptions.cmake

```diff
@@ -0,0 +1,29 @@
+# === BUILD TYPE ===
```

Comment:
Can we name it CommonBuildOptions.cmake?

Reply:
done